lib/util.js

```diff
@@ -476,22 +484,42 @@ module.exports = {
           // TODO: have to add auth here (but first, auth to be put into the openapi tree)
         });
 
-        // recurse over child leaf nodes
-        // and add as children to this folder
-        for (i = 0, requestCount = resource.requests.length; i < requestCount; i++) {
-          itemGroup.items.add(
-            this.convertChildToItemGroup(openapi, resource.requests[i])
-          );
+        // If a folder has only one child which is a folder then we collapsed the child folder
+        // with parent folder.
+        /* eslint-disable */
```

Comment:
this disables all eslint rules. We just want the indentation-limit rule to be disabled.